### PR TITLE
glTFSerializer: Display warning about pivot points not being supported

### PR DESCRIPTION
**Pull request by kcoley**
- Add support for pivot point offsets

### CODE REVIEW

File: serializers/src/glTF/2.0/babylon.glTFExporter.ts [hunk 1]
```diff
@@ -783,8 +783,12 @@ module BABYLON.GLTF2 {
          * @param babylonTransformNode Babylon mesh used as the source for the transformation data
          */
         private setNodeTransformation(node: INode, babylonTransformNode: TransformNode): void {
-            if (!babylonTransformNode.position.equalsToFloats(0, 0, 0)) {
-                node.translation = this._convertToRightHandedSystem ? _GLTFUtilities._GetRightHandedPositionVector3(babylonTransformNode.position).asArray() : babylonTransformNode.position.asArray();
+            const pivotPoint = babylonTransformNode.getPivotPoint();
+            if (!babylonTransformNode.position.equalsToFloats(0, 0, 0) && !pivotPoint.equalsToFloats(0,0,0)) {
```
Comment (deltakosh) [inline serializers/src/glTF/2.0/babylon.glTFExporter.ts]: the pivot point is not exactly a position.
It is more a pre translation (before rotation)
I don't think we can export it as translation

@bghgary is there a notion of Pivot in gltf?

Comment (bghgary) [reply serializers/src/glTF/2.0/babylon.glTFExporter.ts]: Not that I'm aware of.

Comment (deltakosh) [reply serializers/src/glTF/2.0/babylon.glTFExporter.ts]: Ok so we need to chat to see how to correctly support pivot point

**Comment by kcoley** [comment]
@deltakosh @bghgary This update creates a parent node with the pivot translation

**Comment by deltakosh** [comment]
Does it work well with @davrous sample?

**Comment by kcoley** [comment]
Hmm seems to be a bug when I export his scene.  Looking into it.

**Comment by kcoley** [comment]
Changed this PR to display a warning about pivot points not being supported

**Pull request merged**